1-js/02-first-steps/13-while-for/7-list-primes/solution.md

```diff
@@ -16,14 +16,14 @@ The code using a label:
 let n = 10;
 
 nextPrime:
-for (let i = 2; i <= n; i++) { // for each i...
+for (let i = 2; i <= n; i++) { // ամեն i֊ի համար...
```

Comment:
There are a few lines in the beginning of this file that are not translated. I think the pseudo-algorithm above can be left as is but if you want translate it I'm OK with that.